fluent-bit/Dockerfile

```diff
@@ -45,13 +47,17 @@ RUN cmake -DFLB_RELEASE=On \
           -DFLB_SHARED_LIB=Off \
           -DFLB_EXAMPLES=Off \
           -DFLB_HTTP_SERVER=On \
+          -DFLB_IN_EXEC=Off \
           -DFLB_IN_SYSTEMD=On \
           -DFLB_OUT_KAFKA=On \
-          -DFLB_OUT_PGSQL=On ..
+          -DFLB_OUT_PGSQL=On \
+          ..
 
 RUN make -j $(getconf _NPROCESSORS_ONLN)
 RUN install bin/fluent-bit /fluent-bit/bin/
 
+RUN cp -R /tmp/fluent-bit/build/lib/* /tmp/fluent-bit/lib
```

Comment:
Found [another solution](https://github.com/chrkl/third-party-images/commit/3ea5617b206877d2b01493038e99209a506b5bb7) by pointing [FindMonkey.cmake](https://github.com/fluent/fluent-bit/blob/master/cmake/FindMonkey.cmake) to the right directory. But is also fine for me.

Reply:
I took over your solution